guidelines/index.html

```diff
@@ -1876,48 +1876,48 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
     <dl>
-        <dt>Alternative</dt>
-        <dd>Another authentication method that does not rely on a cognitive function test.</dd>
-        <dt>Mechanism</dt>
-      <dd>A <a href="#dfn-mechanism" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-mechanism-13" title="結果を得るためのプロセス又は手法。">mechanism</a> is available to assist the user in completing the cognitive function test.</dd>
-        <dt>Object Recognition</dt>
-        <dd>The cognitive function test is to recognize objects.</dd>
-        <dt>Personal Content</dt>
-      <dd>The cognitive function test is to identify <a href="#dfn-non-text-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-non-text-content-2" title="プログラムによる解釈が可能な文字の並びではないコンテンツ、又は文字の並びが自然言語においても何をも表現していないコンテンツ。">non-text content</a> the user provided to the Web site.</dd>
+        <dt>代替手段がある</dt>
+        <dd>認知機能テストに依存しない別の認証方法がある。</dd>
+        <dt>メカニズムがある</dt>
+        <dd>認知機能テストの完遂を支援する<a href="#dfn-mechanism" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-mechanism-13" title="結果を得るためのプロセス又は手法。">メカニズム</a>が利用できる。</dd>
+        <dt>オブジェクトの認知である</dt>
+        <dd>認知機能テストは、オブジェクトを認知させるものである。</dd>
+        <dt>個人的なコンテンツである</dt>
+        <dd>認知機能テストは、利用者がウェブサイトに提供した<a href="#dfn-non-text-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-non-text-content-2" title="プログラムによる解釈が可能な文字の並びではないコンテンツ、又は文字の並びが自然言語においても何をも表現していないコンテンツ。">非テキストコンテンツ</a>を識別させるものである。</dd>
     </dl>
 
-    <div class="note" role="note" id="issue-container-generatedID-41"><div role="heading" class="note-title marker" id="h-note-41" aria-level="5"><span>注記 1</span></div><p class="">"Object recognition" and "Personal content" may be represented by images, video, or audio.</p></div>
-    <div class="note" role="note" id="issue-container-generatedID-42"><div role="heading" class="note-title marker" id="h-note-42" aria-level="5"><span>注記 2</span></div><div class="">Examples of mechanisms that satisfy this criterion include:<ol>
-            <li>support for password entry by password managers to reduce memory need, and</li>
-            <li>copy and paste to reduce the cognitive burden of re-typing.</li>
+    <div class="note" role="note" id="issue-container-generatedID-41"><div role="heading" class="note-title marker" id="h-note-41" aria-level="5"><span>注記 1</span></div><p class="">「オブジェクトの認知」及び「個人的なコンテンツ」は、画像、映像、又は音声によって表現される可能性がある。</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-42"><div role="heading" class="note-title marker" id="h-note-42" aria-level="5"><span>注記 2</span></div><div class="">この達成基準を満たすメカニズムの例としては、以下が挙げられる。<ol>
+            <li>パスワードマネージャーによるパスワード入力の支援によって、記憶の必要性を軽減する。及び、</li>
```

Comment:
> もしかしたら、この「及び」はトルツメでもよいのかもしれません？

「そして」くらいの意味かと思いますが、敢えて訳さない、というのはありかもしれませんね。

```suggestion
            <li>パスワードマネージャーによるパスワード入力の支援によって、記憶の必要性を軽減する。</li>
```